_posts/2020-10-15-2.0.0-release.md

```diff
@@ -80,6 +80,15 @@ Bugs for JDBC adapters, sort algorithms, and ComplexCopier have been resolved to
 
 ## Python notes
 
+Parquet reading now supports round trip of arbitrarily nested data. Several bug fixes
+for writing nested data and FixedSizeList.  If writing data with these type we recommend validating 
+old data (there is potential some data loss) and upgrade to 2.0.
+
+Extension types now round trip through Parquet.
+
+Conversion of Timezone aware datetimes to and/from pyarrow arrays including pandas now round-trip preserving timezone.  To use the old behavior (e.g. for spark) set the environment variable 
+PYARROW_IGNORE_TIMEZONE
```

Comment:
This has to be set to equal 1 (or some truthy value?) I think

Reply:
yes.  clarified.